src/components/RegistrationPage.jsx

```diff
@@ -0,0 +1,69 @@
+import './Styles.css';
+import React, { useState } from "react";
+import { useHistory } from 'react-router-dom'
+import { InputField } from "./InputField"
+import { Button } from './Button';
+import { registration } from "../app_backend_api/authApi";
+
+export const RegistrationPage = () => {
+    const history = useHistory();
+
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [name, setName] = useState("");
+
+    const registerUser = () => {
+        registration({name, email, password})
+            .then((e) => history.push("/login"))
+            .catch((errorMessage) => console.log(errorMessage));
+    };
+
+    const handleNameChange = (e) => {
+        setName(e)
+    };
+
+    const handleEmailChange = (e) => {
+        setEmail(e)
+    };
+
+    const handlePasswordChange = (e) => {
+        setPassword(e)
+    };
+
+    return (
+        <>
+            <div >
+                Registration page
+                <div>
+                    <div class="Div-inline">{<h6 >Name</h6>}</div>
```

Comment:
rename `class` to `className` everywhere. React does not accept `class` attribute